docs/api/system-preferences.md

```diff
@@ -326,3 +325,20 @@ You can use the `setAppLevelAppearance` API to set this value.
 
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
+
+### `systemPreferences.getMediaAccessStatus(mediaType)` _macOS_
+
+* `mediaType` String - `microphone` or `camera`.
+
+Returns `String` - `not-determined` | `granted` | `denied` | `restricted`
+
+This user consent was not required until macOS 10.14 Mojave, so this method will always return `granted` if your system is running 10.13 High Sierra or lower.
+
+### `systemPreferences.askForMediaAccess(mediaType, askAgain)` _macOS_
+
+* `mediaType` String - the type of media being requested; can be `microphone`, `camera`, or `all` for all media to be requested.
+* `askAgain` Boolean - whether or not the app should request media consent again if it is denied the first time. Defaults to `false`.
+
+Returns `Promise<Boolean>` - A promise that resolves with `true` if consent was granted and `false` if it was denied. If an access request was denied and later is changed through the System Preferences pane, a restart of the app will be required for the new permissions to take effect.
```

Comment:
What happens if invalid arguments are passed?
Please document that the promise will be rejected.